keyboards/pabile/p18/keymaps/default/keymap.c

```diff
@@ -0,0 +1,26 @@
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+	[0] = LAYOUT(
+					         KC_P7,   KC_P8,   KC_P9, KC_PMNS, 
+					         KC_P4,   KC_P5,   KC_P6, KC_PPLS,
+					KC_MUTE, KC_P1,   KC_P2,   KC_P3, KC_TAB,
+					KC_ESC,  KC_DEL, KC_P0, KC_PDOT, KC_PENT)
```

Comment:
```suggestion
    [0] = LAYOUT(
                 KC_P7,   KC_P8,   KC_P9, KC_PMNS, 
                 KC_P4,   KC_P5,   KC_P6, KC_PPLS,
        KC_MUTE, KC_P1,   KC_P2,   KC_P3, KC_TAB,
        KC_ESC,  KC_DEL, KC_P0, KC_PDOT, KC_PENT
    )
```

Reply:
thanks for your changes, drashna. i have accepted and made a commit.